server/player.lua

```diff
@@ -239,12 +314,64 @@ end
 
 exports('RemovePlayerFromJob', RemovePlayerFromJob)
 
+---Removes the player from their current primary gang and adds the player to the new gang
+---@param identifier Source | string
+---@param grade? integer defaults to 0
+---@return boolean success if gang was set
+---@return ErrorResult? errorResult
+function SetGang(identifier, gangName, grade)
+    gangName = gangName:lower()
+    grade = tonumber(grade) or 0
+
+    local gang = GetGang(gangName)
+
+    if not gang then
+        lib.print.error(('cannot set gang. Gang %s does not exist'):format(gangName))
+
+        return false
+    end
+
+    if not gang.grades[grade] then
+        lib.print.error(('cannot set gang. Gang %s does not have grade %s'):format(gangName, grade))
+
+        return false
+    end
+
+    local player = type(identifier) == 'string' and (GetPlayerByCitizenId(identifier) or GetOfflinePlayer(identifier)) or GetPlayer(identifier)
+
+    if setGangReplaces and player.PlayerData.gang.name ~= 'none' then
+        local success, errorResult = RemovePlayerFromGang(player.PlayerData.citizenid, player.PlayerData.gang.name)
+
+        if not success then
+            return false, errorResult
+        end
+    end
+
+    if gangName ~= 'none' then
+        local success, errorResult = AddPlayerToGang(player.PlayerData.citizenid, gangName, grade)
+
+        if not success then
+            return false, errorResult
+        end
+    end
+
+    local success, errorResult = SetPlayerPrimaryGang(player.PlayerData.citizenid, gangName)
+
+    if not success then
+        return false, errorResult
+    end
+
+    return true
```

Comment:
```suggestion
    return SetPlayerPrimaryGang(player.PlayerData.citizenid, gangName)
```
I think we can simplify this section.

Reply:
This was kept this way so that the SetGang player function would still operate the same way.

Reply:
It would still operate the same way, but ok.

Reply:
I just did what I was told. Maybe @Manason can explain more.

Reply:
I agree with d4's suggestion